python/taichi/ad.py

```diff
@@ -1,3 +1,7 @@
+"""Taichi automatic differentiation module.
```

Comment:
Here and below: there's a new line after the first short description. See https://github.com/google/styleguide/blob/gh-pages/pyguide.md#382-modules